### PR TITLE
Add Credo, Overcommit and Asdf tool-versions

### PR DESCRIPTION
**Pull request by eliasousa**

### CODE REVIEW

File: lib/tokio/application.ex [hunk 1]
```diff
@@ -1,5 +1,7 @@
 defmodule Tokio.Application do
+  @moduledoc false
   use Application
+  alias TokioWeb.{Endpoint}
```
Comment (thiagogsr) [inline lib/tokio/application.ex]: You added the alias here but you are not using it in line 16. Also you don't need the brackets if you have only one alias.

So the correct is `alias TokioWeb.Endpoint` and `supervisor(Endpoint, [])`

File: lib/tokio_web/views/error_view.ex [hunk 2]
```diff
@@ -1,5 +1,6 @@
 defmodule TokioWeb.ErrorView do
   use TokioWeb, :view
+  alias Phoenix.{Controller}
```
Comment (thiagogsr) [inline lib/tokio_web/views/error_view.ex]: Also here about the brackets

You can use brackets for `Phoenix.{Alias1, Alias2}` when the aliases are in the same level.

`Phoenix.Module1.SubModule2` can't be merge with `Phoenix.Module2.SubModule3`

File: test/support/channel_case.ex [hunk 3]
```diff
@@ -14,6 +14,7 @@ defmodule TokioWeb.ChannelCase do
   """
 
   use ExUnit.CaseTemplate
+  alias Ecto.{Adapters.SQL.Sandbox}
```
Comment (thiagogsr) [inline test/support/channel_case.ex]: Just `alias Ecto.Adapters.SQL.Sandbox` is enough

File: test/support/conn_case.ex [hunk 4]
```diff
@@ -14,6 +14,8 @@ defmodule TokioWeb.ConnCase do
   """
 
   use ExUnit.CaseTemplate
+  alias Ecto.{Adapters.SQL.Sandbox}
```
Comment (thiagogsr) [inline test/support/conn_case.ex]: Same here

File: test/support/conn_case.ex [hunk 5]
```diff
@@ -14,6 +14,8 @@ defmodule TokioWeb.ConnCase do
...
+  alias Phoenix.{ConnTest}
```
Comment (thiagogsr) [inline test/support/conn_case.ex]: Same here

File: test/support/data_case.ex [hunk 6]
```diff
@@ -13,6 +13,8 @@ defmodule Tokio.DataCase do
   """
 
   use ExUnit.CaseTemplate
+  alias Ecto.{Changeset}
```
Comment (thiagogsr) [inline test/support/data_case.ex]: Same here

File: test/support/data_case.ex [hunk 7]
```diff
@@ -13,6 +13,8 @@ defmodule Tokio.DataCase do
...
+  alias Ecto.{Adapters.SQL.Sandbox}
```
Comment (thiagogsr) [inline test/support/data_case.ex]: Same here

File: lib/tokio/application.ex (hunk 1 above)
Comment (eliasousa) [reply lib/tokio/application.ex]: Nice! done

**Pull request merged**